account_financial_report/wizard/general_ledger_wizard_view.xml

```diff
@@ -91,10 +107,6 @@
                 res_model="general.ledger.report.wizard"
                 src_model="res.partner"
                 view_mode="form"
-                context="{
-                    'default_receivable_accounts_only':1,
-                    'default_payable_accounts_only':1,
```

Comment:
suggestion: you can set Payable, Receivable tags by default so it will be initialized in the same way as before

Reply:
It is already working like this, thanks to the _onchange_partner_id :)